dhcpv6/dhcpv6_test.go

```diff
@@ -133,6 +133,31 @@ func TestFromAndToBytes(t *testing.T) {
 	expected := [][]byte{
 		{01, 0xab, 0xcd, 0xef, 0x00, 0x00, 0x00, 0x00},
 		[]byte("0000\x00\x01\x00\x0e\x00\x01000000000000"),
+		{
```

Comment:
What is this additional test for? Is it covering something we were not before?

Reply:
I wanted to add an example of packet generated by some a device to ensure that it can be parsed correctly.

If you think that this doesn't add value,  I can remove it :-)